app/controllers/time_slot_controller.rb

```diff
@@ -41,10 +41,7 @@ def index
     @ubs = @appointment.try(:ubs)
 
     @time_slots = Ubs.all.where(active: true).each_with_object({}) do |ubs, memo|
```

Comment:
```suggestion
    SLOTS_WINDOW_IN_DAYS = 4
    date_range = build_weekdays_date_range(SLOTS_WINDOW_IN_DAYS)
    @time_slots = Ubs.all.where(active: true).each_with_object({}) do |ubs, memo|
      memo[ubs] = ubs.available_time_slots(date_range, Time.zone.now)
    end
```

Reply:
Was created an attribute `SLOTS_WINDOW_IN_DAYS` (*line 6*)  for `TimeSlotController` class and pass by argument for the `build_weekdays_date_range` (*line 52*) function of `TimeSlotHelper` module.

I guess now is more close to the suggested changes...